Python/Gmeet py Bot/Gmeet_buddy.py

```diff
@@ -3,17 +3,18 @@
 from selenium import webdriver
 import schedule,time
 import keyboard
+import os
 #from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.common.action_chains import ActionChains, Options
 
 
 # joining Credentials
 
-username = str(input('Your Username:'))   # asking for mail id
-password = str(input('Your Password:'))  # asking for password
+username = os.environ.get('Username')       # Fetching mail id from .env
```

Comment:
A `.env` file should never be uploaded, for this reason, `.gitignore` must have ignored it.

Please add a `.env.example` file.

Reply:
That's easy. Let me explain:

Have a .env file in your Directory and add the following:
API_KEY = "XXXXXXXXXXXXXXXXXXXXXXX"
Install a Package named decouple to get the Environment Variables. Import it like this:
from decouple import config

Go to your Main Script and add the following line of code:
api_key = config("API_KEY")

You are now ready to use the API Key in your Code. Do not push the .env in your Code because its not a good practise. Have a .env.example which the User can rename to .env and store their API Key. Have the .env.example for sure.